corehq/motech/repeaters/tests/test_repeater.py

```diff
@@ -1305,6 +1306,10 @@ def setUpClass(cls):
         )
         cls.repeater.save()
 
+    def setUp(self):
+        self.repeater.repeat_records.all().delete()
+        DataSourceUpdate.objects.all().delete()
```

Comment:
This shouldn't be necessary since a `TestCase` performs all db operations in a transaction that is rolled back after each test.

Reply:
Yes, it shouldn't! :thinking: I wonder what glitch in the matrix resulted in me putting that in there. :black_cat: I'll remove that.

Reply:
0713d6980ea